benchmarks/src/jmh/java/software/amazon/jdbc/benchmarks/testplugin/BenchmarkPlugin.java

```diff
@@ -65,6 +64,24 @@ public Connection connect(String driverProtocol, HostSpec hostSpec, Properties p
     return connectFunc.call();
   }
 
+  @Override
+  public Connection forceConnect(String driverProtocol, HostSpec hostSpec, Properties props,
+      boolean isInitialConnection, JdbcCallable<Connection, SQLException> forceConnectFunc)
+      throws SQLException {
+    LOGGER.finer(() -> String.format("forceConnect=''%s''", driverProtocol));
```

Comment:
Should this be driverprotocol here? Do we want to use the hostname for more information?